pages/main/install.hbs

```diff
@@ -77,36 +77,36 @@ slug: install
 </div>
 
 <div class="callout">
-    <h4>What features will and won't be working on my watch with AsteroidOS?</h4>
-    <p>The available features depend on the watch you want to use AsteroidOS on. Take a look at the table in your device's installation page for detailed information on your current model's support.</p>
+    <h4>What features will and won't work on my watch with AsteroidOS?</h4>
+    <p>The available AndroidOS features depend on the watch you want to use. Review the table in your device's installation page for detailed support information.</p>
```

Comment:
AsteroidOS* :)

Reply:
Thanks - I'm glad it's helpful! I may have some upcoming free time to review more of the website. I've been looking for a good candidate for a modern watch on which to test AsteroidOS, so I probably won't be of much help with detailed documentation until I buy one and try it out.